source/fundamentals/bson.txt

```diff
@@ -210,6 +210,29 @@ Driver will marshal structs using the following rules:
       - Includes an empty ``lastname`` field
       - Stores the ``Address`` field as a nested value
 
+BSON Options
+------------
+
+You can set ``BSONOptions`` to specify how
+the driver marshals and unmarshals BSON on any level, from
+``collection`` to ``Client``.
```

Comment:
Issue: I'm not sure what "any level" refers to. How many are there?

Suggestion: Can we be more specific about these levels?